README.md

```diff
@@ -87,6 +87,7 @@ $ git clone https://github.com/borgbase/ansible-role-borgbackup.git roles/ansibl
 - `borg_exclude_from`: Read exclude patterns from one or more separate named files, one pattern per line.
 - `borg_exclude_patterns`: Paths or patterns to exclude from backup. See [official documentation](https://borgbackup.readthedocs.io/en/stable/usage/help.html#borg-help-patterns) for more.
 - `borg_install_method`: By default `pip` is used to install borgmatic. To install via your distributions package manager set this to `package` and (if needed) overwrite the `borg_distro_packages` variable to contain your distributions package names required to install borgmatic. Note that many distributions ship outdated versions of borgbackup and borgmatic; use at your own risk.
+- `borg_require_epel`: When using `borg_install_method: package` on RHEL-based distributions, the EPEL repo is required. To disable the check (e.g. when using a custom mirror instead of the epel-release package), set this to `false`.
```

Comment:
How about explaining the default value? Like "Defaults to `true` on Enterprise Linux-based distros, except Fedora."

Reply:
Sounds good, thanks, I will add something - let me think about the wording. I don't think Fedora is an "Enterprise Linux-based distro" (it's the other way around, EL is based on Fedora), so maybe just "Defaults to `true` on Enterprise Linux-based distros"? Or something about the `RedHat` OS family except Fedora. I wish there was a better Ansible Fact for EL distros, I was kind of shocked when I noticed Fedora was in there.